GOVERNANCE.md

```diff
@@ -116,6 +116,7 @@ Certain project decisions require a vote. These include:
 
 - Governance changes: simple majority (over 50%) conducted via GitHub PR approval.
 - Core contributor membership: overwhelming majority (over 70%) conducted by privately messaging a steward. Funneling both assenting and dissenting votes directly through stewards allows for anonymity when discussing the merits of a potential contributor.
+- Committer contributor membership: discussion conducted via a thread with all existing L2 contributors. Thread will be deleted once a decision is made.
```

Comment:
- mention simple vs. overwhelming majority
- actually, everything added here feels relevant to both membership voting, not just committer. Does it make more sense to add this to the existing bullet point as "~core~ contributor membership" and then also remove the point about private messaging for both? I'd be +1 on that change.